src/app/pipes/truncate-decimal.pipe.ts

```diff
@@ -0,0 +1,16 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+ name: 'truncateDecimal'
```

Comment:
Why create a new pipe when you can use the built-in [DecimalPipe](https://angular.io/api/common/DecimalPipe)? This is already used in many places in the project

Reply:
Yeah just noticed that. I have changed that to the built-in Decimal pipe and pushed the changes